test/cmocka/src/audio/eq_fir/cmocka_fir_in.h

```diff
@@ -0,0 +1,2409 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2022 Intel Corporation. All rights reserved.
+ */
+
+#define FIR_2CH_LENGTH	19200
+
+unsigned int fir_2ch[] = {
```

Comment:
Could this be the same input as IIR is using ../../../include/cmocka_chirp_2ch.h ?

Reply:
sure. And what's the ref output ? If we uses pass-through coef, this is not a problem

Reply:
I'm doing a script that exports all needed ref and coef blob files. Then it will be easy to add more tests and chang them.

I think the cause of fail with the current error criteria is quantization of FIR coefficients to 16 bit. Need to generate the reference with decoded coefficients from the blob. I'm working on it now.